src/components/PlayerButtons.js

```diff
@@ -37,89 +36,66 @@ export const ForwardButton = (props) => {
         size={25}
         color="#fff"
       />
-      );
+    );
   }
 
   return forwardButton;
 };
 
-
-export const BackwardButton = props => (
+export const BackwardButton = props =>
   <FontAwesome
     onPress={props.goBackward}
     style={Styles.back}
     name="backward"
     size={25}
     color="#fff"
-  />
-    );
+  />;
 
-export const VolumeButton = props => (
+export const VolumeButton = props =>
   <FontAwesome
     onPress={props.toggleVolume}
     style={Styles.volume}
     name={props.muted ? 'volume-off' : 'volume-up'}
     size={18}
     color="#fff"
-  />
-    );
+  />;
 
-export const ShuffleButton = props => (
-  <FontAwesome
+const ShuffleIcon = styled(FontAwesome)`
+  margin-top: 26px;
+  `;
+
+export const ShuffleButton = props =>
+  <ShuffleIcon
     onPress={props.toggleShuffle}
-    style={Styles.shuffle}
     name="random"
     size={18}
     color={props.shuffle ? '#f62976' : '#fff'}
-  />
-    );
+  />;
 
-export const DownloadButton = (props) => {
-  if (!props.download || props.downloading) {
-    return (
-      <FontAwesome
-        style={Styles.downloadButton}
-        name="download"
-        size={25}
-        color="#333"
-      />
-    );
-  }
+export const SongSlider = (props) => {
+  const currentTime = Utils.secondTime(props.currentTime) || '00:00';
+  const durationTime = Utils.secondTime(props.songDuration) || '00:00';
```

Comment:
these static string ('00:00') might be moved to a class that holds such constants too